plugins/in_http/http_prot.c

```diff
@@ -493,6 +496,12 @@ static int process_payload(struct flb_http *ctx, struct http_conn *conn,
 {
     int type = -1;
     struct mk_http_header *header;
+    char *uncompressed_data;
+    size_t uncompressed_size = 0;
+
+    /* used when checking content-encoding */
+    bool gzip_compressed = false;
```

Comment:
Still bool type is existing. Could you use int type variable which is initialized with`FLB_FALSE` here?

Reply:
Sorry, missed that one. Fixed now.